src/library/Database.js

```diff
@@ -1,15 +1,8 @@
 import PouchDb from 'pouchdb'
 
 export class Database {
-  static _instance = null
```

Comment:
I changed the way that you load the database. I'm not sure if you have a better way to do this.